packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "12.1.0",
+  "version": "12.1.1",
```

Comment:
I don't think we need to bump the version since publishing these changes won't really effect anything.